device/delta/x86_64-delta_ag9032v2a-r0/sensors.conf

```diff
@@ -0,0 +1,4 @@
+# libsensors configuration file for AG9032v2a
+# ------------------------------------------------
+#
+
```

Comment:
how come this file is empty?

Reply:
Those devices is handled by BMC and this file is not necessary. I will remove it